src/routing/tests/mod.rs

```diff
@@ -1142,6 +1210,30 @@ fn create_section(
     Ok((section, section_key_share))
 }
 
+// Create a `Vote::Online` whose consensus handling triggers relocation of a node with the given age.
+// NOTE: recommended to call this with low `age` (4 or 5), otherwise it might take very long time
```

Comment:
better to explain the longer time for a higher age is due to make signature having more trailing zeroes?